tests/FunctionalJavascript/DefaultContentOwnerTest.php

```diff
@@ -58,4 +58,48 @@ public function testDefaultContentOwnerValues() {
     $assert_session->fieldValueEquals('oe_content_content_owner[1][target_id]', 'Directorate-General for Budget (http://publications.europa.eu/resource/authority/corporate-body/BUDG)');
   }
 
+  /**
```

Comment:
I see you added 2 new tests, but let's merge the tests together no? We don't need to run this thing 3 times, it's very slow. So let's merge all tests in one.

Reply:
I collected all tests into one test.